fixtures/american-dj/quad-phase-hp.json

```diff
@@ -0,0 +1,168 @@
+{
+  "name": "Quad Phase HP",
+  "shortName": "QPHP",
+  "categories": ["Flower"],
+  "meta": {
+    "authors": ["FloEdelmann"],
+    "createDate": "2017-04-09",
+    "lastModifyDate": "2017-04-09"
+  },
+  "manualURL": "http://intranet.americandj.com/ItemRelatedFiles/6976/quad_phase_hp.pdf",
+  "physical": {
+    "dimensions": [312, 198, 279],
```

Comment:
This should be width × height × depth, but the order from the manual that you used is 12.5”(L) x 8”(W) x 11”(H).